exec/src/klio_exec/commands/run.py

```diff
@@ -18,9 +18,11 @@
 import logging
 import os
 import re
+import time
 
 import apache_beam as beam
 from apache_beam.options import pipeline_options
+from apache_beam.runners.runner import PipelineResult, PipelineState
```

Comment:
Nitpick: we follow the google style of code, which means we import modules, not specific classes/objects from modules. Mind changing this to `from apache_beam.runners import runner`?

Reply:
👍🏾